release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1376,22 +1376,22 @@ In the table below, features are marked with the following statuses:
 |CSI volume expansion
 |TP
 |TP
-|
+|GA
 
 |CSI AliCloud Disk Driver Operator
 |-
 |GA
 |GA
 
 |CSI Azure Disk Driver Operator
-|TP
+|-
 |GA
 |GA
 
 |CSI Azure File Driver Operator
-|-
 |TP
-|
+|GA
```

Comment:
I don't think this is right. This is listed as newly TP in the 4.10 RNs. So I think it should be `- TP GA` (as long as it really did go GA in 4.11)